pt-PT/Universally Preferable Behaviour/p02_ch03_the_second_test_murder.md

```diff
@@ -1,39 +1,39 @@
 
-If murder is morally good, then clearly *refraining* from murder is immoral. Thus the only chance that Bob and Doug have to be moral is in the instant that they simultaneously murder each other. Physically, this is impossible of course – if they both stand and grip each other’s throats, they will never succumb to strangulation at exactly the same moment. If Bob dies first, his grip on Doug’s throat will loosen, thus condemning Doug to the status of *immorality* until such time as he can find another victim. Because Bob dies first – and thus cannot continue to try murdering Doug – Bob’s death renders him more immoral than Doug’s murder.
+Se o assassinato é moralmente bom, então claramente *abster-se* de assassinar é imoral. Assim, a única hipótese que o Bob e o Doug têm de ser moral é no instante em que eles simultaneamente se assassinarem um ao outro. Fisicamente, isto é impossível como é obvio – se os dois estão de pé e ambos agarrarem o pescoço um do outro, eles nunca vão sucumbir por estrangulamento ao mesmo tempo. Se o Bob morrer primeiro, o aperto na garganta do Doug será perdido, assim condenando Doug ao status de *imoralidade*, até ao momento que ele encontre outra vítima. Como o Bob morre primeiro - e assim não pode continuar a tentar matar o Doug - a morte do Bob rende-lhe mais imoral do que o assassinato de Doug.
 
-Intuitively, we fully recognise the insanity of the moral proposition that murder is good. Logically, we know that the proposition is incorrect because if it is true, it is impossible for two men in a room to both be moral at the same time. Morality, like health, cannot be considered a mere “snapshot,” but must be a process, or a continuum. The UPB framework confirms that Bob cannot be “evil” *while* he is strangling Doug, and then achieve the pinnacle of moral virtue the moment that he kills Doug – and then revert immediately back to a state of evil. Moral propositions must be universal, and independent of time and place. The proposition that murder is moral fails this requirement at every level, and so is not valid.
+Intuitivamente, nós reconhecemos plenamente a insanidade de uma proposição moral de que o assassinato é bom. Logicamente, nós sabemos que esta proposição é incorreta porque se fosse verdade, seria impossível para dois homens numa sala serem ambos morais ao mesmo tempo. A moralidade, como a saúde, não podem ser consideradas meramente "instantâneas," mas precisam de ser um processo, ou um continuum. A estrutura do UPB confirma que o Bob não pode ser "mau" *enquanto* ele se encontra a estrangular o Doug, e então alcançar o ápice da virtude moral no momento em que ele mata o Doug - e depois reverter imediatamente para o estado de mau. As proposições morais têm que ser universais, e independentes do tempo e lugar. A proposição de que o assassinato é moral falha neste requerimento a todos os níveis, e então não é válida.
```

Comment:
You used "se fosse verdade", which is not easy for an automated translation. So I do believe you're doing the translation by hand

Reply:
My mistake, I'm going to change it. Regarding the translations I do manually without the use of translation machines, I only use them when an unknown word appears.